actionpack/lib/action_controller/metal/content_security_policy.rb

```diff
@@ -28,6 +51,18 @@ def content_security_policy(enabled = true, **options, &block)
         end
       end
 
+      # Allows overriding the global `content_security_policy_report_only`
+      # in the controller:
+      #
+      #   class PostsController < ApplicationController
+      #     content_security_policy_report_only only: :index
+      #   end
+      #
+      # Pass +false+ to disable reporting:
+      #
+      #   class PostsController < ApplicationController
+      #     content_security_policy_report_only false, only: :index
+      #   end
```

Comment:
Omitting the first arg in the first example made me look twice.  I see that `true` is the default value, and I do think that `content_security_policy_report_only only: :index` reads nicer than `content_security_policy_report_only true, only: :index`.  But I wanted to note that the difference between the examples was surprising at first.

Reply:
Yes, that is one of the reasons I wanted to document these methods.
https://github.com/rails/rails/pull/44015 documents some other API suprises.